shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1238,13 +1240,24 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
     }
 
     @Data
+    @AllArgsConstructor
```

Comment:
This is included in `@Data`. Do you need it for something specific?

Reply:
According to the documentation, `@Data` includes `@RequiredArgsConstructor` instead of `@AllArgsConstructor`, right? (I don't think it matters much in this case, though). In any case, this is needed because the behavior of `@Data` seems to create the constructor if no constructor is yet defined. As you can see, I have added a custom constructor (without `attributeUpdates`) for comparability. In this case, the "all args" constructor does not get generated (I tested that by removing the custom constructor, and in that case the "all args" constructor gets generated). Apparently, by explicitely adding the annotation `@AllArgsConstructor`, Lombok generates the constructor with all the arguments, irrespective of other custom constructors defined.